bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WpWpJJ_EWK-QCD/WWjj_SS_ewk_qcd_run_card.dat

```diff
@@ -0,0 +1,270 @@
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	=  run_tag  ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  True	=  gridpack   !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  10000	=  nevents  ! Number of unweighted events requested 
+  0	=  iseed    ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  1	=  lpp1     ! beam 1 type 
+  1	=  lpp2     ! beam 2 type
+  6800.0	=  ebeam1   ! beam 1 total energy in GeV
+  6800.0	=  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0	=  polbeam1  ! beam polarization for beam 1
+  0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf    = pdlabel     ! PDF set                                  
+  320900 = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
change to "$DEFAULT_PDF_SETS = lhaid"

Reply:
Hi, @menglu21  
I have modified the cards according to your suggestions. Please check.